src/deployments/inputs/HorizonInput.sol

```diff
@@ -0,0 +1,68 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+import './MarketInput.sol';
+
+contract HorizonInput is MarketInput {
+  address public constant ETH_USD_PRICE_FEED = 0x5f4eC3Df9cbd43714FE2740f5E3616155c5b8419; // chainlink price feed
```

Comment:
confirmed we don't want to go with svr feed?
https://docs.chain.link/data-feeds/price-feeds/addresses?page=1&testnetPage=1&search=eth%2Fusd